gait3d/gaitanalysis_top.py

```diff
@@ -0,0 +1,979 @@
+import pandas as pd
```

Comment:
Should probably be deleted, since it is an file, that's no longer used, as well as the old walkthrough.ipynb

Reply:
deleted